scripts/install/installer

```diff
@@ -117,11 +123,23 @@ def get_pnodes(config, versions=None, roles=None):
             pnodes.append(PNode(config, pnode['name'], versions))
     return pnodes
 
+def local_execute(*args, **kwargs):
+    kwargs['check'] = kwargs.get('check', True)
+    kwargs['stdout'] = kwargs.get('stdout', subprocess.PIPE)
+    result = subprocess.run(*args, **kwargs).stdout.decode('utf-8')
+    return result
+
 def set_cgroup(node, cgroup, cpu_set, memory):
-    node.core.dir_ensure('/sys/fs/cgroup/memory/%s' % cgroup)
-    node.core.dir_ensure('/sys/fs/cgroup/cpuset/%s' % cgroup)
-    node.core.run('echo %s > /sys/fs/cgroup/cpuset/%s/cpuset.cpus' % (cpu_set, cgroup))
-    node.core.run('echo %sG > /sys/fs/cgroup/memory/%s/memory.limit_in_bytes' % (memory, cgroup))
+    mem_path = '/sys/fs/cgroup/memory/{}'.format(cgroup)
+    cpu_path = '/sys/fs/cgroup/cpuset/{}'.format(cgroup)
+    command = """
+    set -e
+    mkdir -p {mem}
+    mkdir -p {cpu}
+    """.format(mem=mem_path, cpu=cpu_path)
+    node.run(command)
+    node.write_file('{}/cpuset.cpus'.format(cpu_path), cpu_set)
+    node.write_file('{}/memory.limit_in_bytes'.format(mem_path), memory)
```

Comment:
cpu-set and memory might be int write_file will crash if they are we need to str those